src/conversions.jl

```diff
@@ -667,12 +667,12 @@ struct Reverse{T}
     data::T
 end
 
-function convert_attribute(r::Reverse, ::key"colormap")
-    reverse(to_colormap(r.data))
+function convert_attribute(r::Reverse, ::key"colormap", n::Integer = 20)
```

Comment:
Is this also needed? This can't really work I think....

Reply:
why not?

Reply:
Huh, maybe it's the right way :D